### PR TITLE
rkn.gov.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by jeremyn** [comment]
There are a lot of other subdomains for this ruleset, can you please add those? See [here](https://www.google.com/search?q=site%3A*.rkn.gov.ru&ie=utf-8&oe=utf-8#q=site:*.rkn.gov.ru+-site:eng.rkn.gov.ru+-site:www.rkn.gov.ru+-site:reestr-svyaz.rkn.gov.ru+-site:pd.rkn.gov.ru+-site:eais.rkn.gov.ru+-site:nap.rkn.gov.ru+-site:398-fz.rkn.gov.ru+-site:blocklist.rkn.gov.ru+-site:97-fz.rkn.gov.ru).

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/Rkn.gov.ru.xml [hunk 1]
```diff
@@ -1,14 +1,94 @@
...
+	<target host="eng.rkn.gov.ru" />
+	<target host="mail.rkn.gov.ru" />
+	<target host="nap.rkn.gov.ru" />
+	<target host="pd.rkn.gov.ru" />
+	<target host="reestr-svyaz.rkn.gov.ru" />
+	<target host="vigruzki.rkn.gov.ru" />
+
+	<target host="www.rkn.gov.ru" />
+
+	<rule from="^http://www\.rkn\.gov\.ru/"
```
Comment (J0WI) [inline src/chrome/content/rules/Rkn.gov.ru.xml]: Please give a hint about the issue.

**Comment by J0WI** [comment]
This might be replaced by #9480

**Pull request closed without merging**